monitor/ConsoleReporter.js

```diff
@@ -72,16 +72,20 @@ define(function(require) {
 				// IE8 has console.log and JSON, so we can make a
 				// reasonably useful warn() from those.
 				// Credit to webpro (https://github.com/webpro) for this idea
-				if (typeof localConsole.log ==='function'
-					&& typeof JSON !== 'undefined') {
-					log = warn = function (x) {
-						if(typeof x !== 'string') {
+				// typeof localConsole.log will return 'object' in IE8, so can't test it with === 'function'
+				// Since this is more of a corner case for IE8, I'm ok to check it with !== 'undefined' to reduce complexity
+				if (typeof localConsole.log !== 'undefined' && typeof JSON !== 'undefined') {
```

Comment:
@sompylasar Do you have a link to some info on that?  I'm not aware of a case where IE would throw on `console` property access (unless of course, `console` is undefined, which is already covered).  I certainly wouldn't put it past IE doing something crazy :D but I'd like to verify.  Thanks!